presto-docs/src/main/sphinx/release/release-0.241.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.241
+=============
+
+General Changes
+_______________
+* Fix incorrect results from function classification_precision() introduced in release 0.239
+* Add dynamic filtering and bucket pruning support for inner join and semi join, which can avoid full table scan on the probe side of broadcast joins or collocated joins when the build side is small. This can be enabled with the ``experimental.enable-dynamic-filtering`` configuration property and ``enable_dynamic_filtering`` system session property. The build side join key space size can be tuned by the ``experimental.dynamic-filtering-max-per-driver-row-count`` and ``experimental.dynamic-filtering-max-per-driver-size`` configuration properties and ``dynamic_filtering_max_per_driver_row_count`` and ``dynamic_filtering_max_per_driver_size`` session properties. Currently, only Hive connector can benefit from the feature.
+* Add new warning message for ``UNION`` queries without ``ALL`` or ``DISTINCT`` keywords.
+* Downgrade the ZSTD JNI compressor version to resolve the frequent excessive GC events introduced in version 0.238.
+
+Security Changes
+________________
+* Implement REST endpoint authorization in Presto. See :doc:`/security/authorization`.
+
+JDBC Changes
+____________
+* Add support for JDBC ``ResultSet.getStatement``.
+* Add support for Oracle JDBC connections.
+
+Hive Changes
+____________
+* Fix several memory counting bugs in ``OrcRecordReader`` and ``StreamReader``'s.
```

Comment:
counting -> accounting reads better?

Reply:
What is the trailing `'s`, remove?

Reply:
> What is the trailing `'s`, remove?

`StreamReader` is an interface. There're several implementations and the `'s` is plural form. I used plural form because the fixes were on several StreamReaders. Let's consult @aweisberg who is native speaker and runs the blog. Ariel, which way is better according to your opinion?

Reply:
I removed the `'` and kept the `s`. So now it will look like ``StreamReader``s